src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacet.ts

```diff
@@ -197,6 +197,17 @@ export class DynamicHierarchicalFacet extends Component implements IDynamicHiera
      * **Default:** `undefined` and the hierarchical facet does not depend on any other facet to be displayed.
      */
     dependsOn: ComponentOptions.buildStringOption(),
+
+    /**
+     * TODO: document option
```

Comment:
I'm actually really interested in reading what this parameter does 😂 .

I'm not sure I understand exactly...

Reply:
It basically filters out Folded documents from the facet's count